nussl/core/audio_signal.py

```diff
@@ -201,6 +201,7 @@ def __init__(self, path_to_input_file=None, audio_data_array=None, stft=None, la
         # Effects
         self._ffmpeg_effects_chain = []
         self._sox_effects_chain = []
+        self.effects_applied = []
```

Comment:
Awesome thanks! This needs to documented now in the Attributes section (here: https://github.com/nussl/nussl/blob/ce3f8792dcfe10c28c2ba2b5a8c393580137adec/nussl/core/audio_signal.py#L159)

Reply:
Also add a reference to the explaining docs in `apply_effects()` ("for more info see...")